<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteEditor</class>
 <widget class="QMainWindow" name="PaletteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palette Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_Settings">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_Palette">
         <property name="text">
          <string>Palette</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_PaletteId">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_BitDepth">
         <property name="text">
          <string>Bit Depth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bit_depth_15">
         <property name="text">
          <string>15</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bit_depth_24">
         <property name="text">
          <string>24</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea_Colors">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaContents_Colors">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>883</width>
         <height>784</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="layout_Colors">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionImport_Palette"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ShortcutContext::WindowShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::Priority::NormalPriority</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionImport_Palette">
   <property name="text">
    <string>Import Palette</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
